app/javascript/article-form/articleForm.jsx

```diff
@@ -47,7 +47,7 @@ export default class ArticleForm extends Component {
     version: PropTypes.string.isRequired,
     article: PropTypes.string.isRequired,
     organizations: PropTypes.string,
-    logoSvg: PropTypes.string.isRequired,
```

Comment:
**(question)**: Was this maybe not named this way to make it easier to differentiate between logo_png and logo_svg that come from the site config (w/o having to trace back to the start)?

Reply:
I don't know the purpose of its original name, but I've decided to rename it since it will no longer be only about logo_svg. The logic we're about to ship soon is following:

- `logo_svg` is not longer required field.
- if you upload `logo_svg`, sweet - this gonna be your loog.
- if you don't upload `logo_svg` we will use site name (actual text label) as placeholder.

And because this can either be `SVG` element **OR just a text label**, the name "_logoSvg_" no longer makes sense.